hw/top_earlgrey/data/clocks.xdc

```diff
@@ -0,0 +1,23 @@
+## Copyright lowRISC contributors.
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.
+## SPDX-License-Identifier: Apache-2.0
+
+## Clock Signal
+create_clock -add -name sys_clk_pin -period 10.00 -waveform {0 5} [get_ports IO_CLK]
+
+## Clock Domain Crossings
+set clks_50_unbuf [get_clocks -of_objects [get_pin clkgen/pll/CLKOUT0]]
+set clks_48_unbuf [get_clocks -of_objects [get_pin clkgen/pll/CLKOUT1]]
+
+## Divided clock
+## This is not really recommended per Vivado's guidelines, but hopefully these clocks are slow enough and their
+## destination flops few enough.
+
+create_generated_clock -name clk_io_div2 -source [get_pin clkgen/pll/CLKOUT0] -divide_by 2 \
```

Comment:
sorry, I should have mentioned this before. the same could be done here as well:
`[get_pin top_earlgrey/u_clkmgr/u_io_div2_div/gen_div2.u_div2/gen_generic.u_impl_generic/q_o_reg[0]/C]`

Reply:
is there a big difference doing it here relative to the source clock though..?

Reply:
oh I only mentioned it for consistency reasons (i.e. either align both statements with source clock or with `C` inputs). 

technically, the phase offset (due to clock routing delay) should not matter in this case since the derived clock groups are defined asynchronous wrt to the source clock...